src/components/GPUProviders.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ClusterOverviewDetailsItem } from '@openshift-console/dynamic-plugin-sdk';
+import { useTranslation } from 'react-i18next';
+
+import { useAllGpuProviders } from '../resources/nodes';
+import { Dash } from './Dash';
+
+const GPUProviders: React.FC = () => {
+  const { t } = useTranslation('plugin__console-plugin-nvidia-gpu');
+  const [providers, loaded, loadError] = useAllGpuProviders();
+
+  if (loadError) {
+    return <Dash />;
+  }
+
+  let result = t('No GPUs currently detected');
+  if (providers.nvidia) {
+    if (providers.amd) {
+      if (providers.other) {
+        result = t('NVIDIA, AMD and others');
+      } else {
+        result = t('NVIDIA and AMD');
+      }
+    } else {
+      if (providers.other) {
+        result = t('NVIDIA and others');
+      } else {
+        result = t('NVIDIA');
+      }
+    }
+  } else {
+    if (providers.amd) {
+      if (providers.other) {
+        result = t('AMD and others');
+      } else {
+        result = t('AMD');
+      }
+    } else {
+      if (providers.other) {
+        result = t('others');
+      }
+      // else no GPU
+    }
+  }
+
+  return (
+    <ClusterOverviewDetailsItem
```

Comment:
The name indicates that this `ClusterOverviewDetailsItem` is specific for Cluster Overview but I think we want shared component for all Overview pages -  so I propose to rename it to `OverviewDetailsItem`

Reply:
ok, renamed